alma/controllers/hook/GetContentHookController.php

```diff
@@ -87,7 +87,7 @@ public function processConfiguration()
         if ($liveKey !== Settings::getLiveKey()) {
             $shareOfCheckoutHelper->resetShareOfCheckoutConsent();
         } else {
-            $shareOfCheckoutHelper->handleCheckoutConsent('ALMA_ACTIVATE_SHARE_OF_CHECKOUT_ON');
+            $shareOfCheckoutHelper->handleCheckoutConsent(ShareOfCheckoutAdminFormBuilder::ALMA_ACTIVATE_SHARE_OF_CHECKOUT . '_ON');
```

Comment:
why is there a constant with string concatenation here?

Reply:
@syjust-alma Because I wanted to use the constant but the builder form Prestashop add '_ON' at the end of the value of inputs switch.
Do you prefer I create an other constant ?

Reply:
just a `// code comment` about that please